README.md

```diff
@@ -1,14 +1,12 @@
 
 # Linus' Kitchen
 
-[![Circle CI](https://circleci.com/gh/tknerr/linus-kitchen/tree/master.svg?style=shield)](https://circleci.com/gh/tknerr/linus-kitchen/tree/master)
+[![Build Status](https://travis-ci.com/fasmat/linus-kitchen.svg?branch=customize-installed-applications)](https://travis-ci.com/fasmat/linus-kitchen)
```

Comment:
Curious on why you replaced circleci with travisci -- just for personal preference, or other reasons involved?

Reply:
I'm not sure why I decided to replace one with the other. If I remember correctly CircleCI was giving me warnings that an upgrade is required and it seemed more complicated than just to switch to travis.